xonsh/base_shell.py

```diff
@@ -15,11 +13,10 @@
 
 
 class TeeOut(object):
-    """Tees stdout into the original sys.stdout and another buffer instance that is 
-    provided.
-    """
+    """Tees stdout into the original sys.stdout and another buffer."""
 
     def __init__(self, buf, *args, **kwargs):
+        _, _ = args, kwargs
```

Comment:
This seems like the linter is just being silly. `*args, **kwargs` are there to future proof a function or method and this discourages that.  If you really want, I guess the names could be changed to `*_, **__` or `*_args, **_kwargs` but I would argue that is less readable. .